src/visGeometry/index.ts

```diff
@@ -802,7 +802,7 @@ class VisGeometry {
                         visAgent.agentData.instanceId,
                         visAgent.signedTypeId(),
                         // a scale value for LODs
-                        (index + 1) * 0.25
+                        0.25 + index * 0.25
```

Comment:
this isn't really a change but it makes it a little easier to tweak by starting with a level 0 radius of 0.25 and then an additive term based on LOD.